plugin/storage/integration/grpc_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"os"
+	"testing"
+
+	"github.com/jaegertracing/jaeger/pkg/config"
+	"github.com/uber/jaeger-lib/metrics"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+	"github.com/jaegertracing/jaeger/plugin/storage/grpc"
+)
+
+type GRPCStorageIntegrationTestSuite struct {
+	StorageIntegration
+	logger *zap.Logger
+}
+
+func (s *GRPCStorageIntegrationTestSuite) initialize() error {
+	s.logger, _ = testutils.NewLogger()
+	gopath := os.Getenv("GOPATH")
+	path := gopath + "/src/github.com/jaegertracing/jaeger/examples/memstore-plugin/memstore-plugin"
```

Comment:
This hardcoded path feels flakey and requires that the memstore-plugin is already built. I don't really see any better way of doing it though.

Reply:
https://github.com/jaegertracing/jaeger/issues/1516